mavros_local_position_pub: '/setpoint_position/local'
mavros_velocity_pub : '/setpoint_velocity/cmd_vel'
mavros_local_atual :  '/local_position/pose'
mavros_state_sub :    '/state'
mavros_arm :          '/cmd/arming'
mavros_set_mode :     '/set_mode'
mavros_battery_sub :  '/battery'
extended_state_sub:   '/extended_state'
mavros_global_position_sub: '/global_position/global'
mavros_pose_target_sub:  '/setpoint_raw/local'
mavros_set_global_pub: '/setpoint_position/global'